packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -691,4 +691,34 @@ Terra.describeViewports('DateTimePicker', ['tiny', 'large'], () => {
 
     Terra.it.validatesElement();
   });
+
+  describe('With timezone', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-with-timezone'));
+
+    Terra.it.matchesScreenshot();
+  });
+
+  describe('With timezone missing hour', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-with-timezone-missing-hour');
+      browser.refresh();
+      Terra.hideInputCaret('input[name="terra-time-minute-input"]');
+      browser.setValue('input[name="terra-date-input"]', '03/11/2018');
+      browser.setValue('input[name="terra-time-hour-input"]', '02');
+      browser.setValue('input[name="terra-time-minute-input"]', '30');
```

Comment:
Can we move this to its own `it` block instead of the before hook?

Reply:
updated in this commit 2795049548a9f35213a3e290861bea40d3ff2ffe